rubygem-smart_proxy_remote_execution_ssh/rubygem-smart_proxy_remote_execution_ssh.spec

```diff
@@ -69,13 +71,23 @@ cp -pa .%{gem_instdir}/bundler.plugins.d/remote_execution_ssh.rb %{buildroot}%{f
 mkdir -p  %{buildroot}%{foreman_proxy_settingsd_dir}
 cp -pa .%{gem_instdir}/settings.d/remote_execution_ssh.yml.example %{buildroot}%{foreman_proxy_settingsd_dir}/remote_execution_ssh.yml
 
+mkdir -p %{buildroot}/opt/theforeman/tfm/root/%{_datadir}/smart_proxy_dynflow_core/bundler.d
+cat <<EOF > %{buildroot}/opt/theforeman/tfm/root/%{_datadir}/smart_proxy_dynflow_core/bundler.d/foreman_remote_execution_core.rb
+gem 'foreman_remote_execution_core'
+EOF
+
 %files
 %dir %{gem_instdir}
 %{gem_instdir}/lib
 %{gem_instdir}/settings.d
 %{foreman_proxy_bundlerd_dir}/remote_execution_ssh.rb
 %config %{foreman_proxy_settingsd_dir}/remote_execution_ssh.yml
 %doc %{gem_instdir}/LICENSE
+%if 0%{?fedora}
+%exclude /opt/theforeman/tfm/root/%{_datadir}/smart_proxy_dynflow_core/bundler.d/foreman_remote_execution_core.rb
```

Comment:
Is %exclude correct here, shouldn't it just be a different path rather than being packaged differently? The directory exists in the Fedora package.